### DISCUSSION TITLE
Version 2.0.0 pre-release 10

### DISCUSSION THREAD

**Discussion by samerton** [NamelessMC/Nameless#2293] [Releases]
- Add ability to upload favicon
- Add Smarty security policy
- Add support for environment variable based debugging
- Auto detect user timezone when registering
- Avatar rework and add Crafthead support
- Captcha rework including Recaptcha v3 invisible support
- Ensure cookie_secure is set in PHP config
- Error handling revamp
- Increase label HTML length
- Move captcha/https/www configuration to config.php
- Placeholder API integration
- Remove unsafe tags when editing posts
- Validation revamp
- API updates - including group sync fixes
- Default Revamp and Default panel template fixes
- htaccess fixes and improvements
- MCAssoc fixes
- Server status widget fixes
- Translation updates
- PHP 8 improvements
- Full changeset https://github.com/NamelessMC/Nameless/compare/v2.0.0-pr9...v2.0.0-pr10

<hr /><em>This discussion was created from the release <a href='https://github.com/NamelessMC/Nameless/releases/tag/v2.0.0-pr10'>Version 2.0.0 pre-release 10</a>.</em>

**Comment by colocated** [comment]
# poggers gaming

**Comment by finnie2006** [comment]
is php 8 stable enough?

**Reply by Derkades** [reply]
Not yet
